core/src/main/java/io/neow3j/transaction/Transaction.java

```diff
@@ -316,17 +321,20 @@ public Builder validUntilBlock(long blockNr) {
         }
 
         /**
-         * Sets the sender of this transaction.
-         * <p>
-         * The sender's account will be charged with the network and system fees.
+         * Adds the given signers to this transaction.
          * <p>
-         * This property is <b>mandatory</b>.
+         * The first signer will be used as the sender of this transaction, i.e. the payer of the
+         * transaction fees.
          *
-         * @param sender The sender account's script hash.
+         * @param signers Signers for this transaction.
          * @return this builder.
          */
-        public Builder sender(ScriptHash sender) {
-            this.sender = sender;
+        public Builder signers(Signer... signers) {
+            if (containsDuplicateSigners(signers)) {
+                throw new TransactionConfigurationException("Can't add multiple signers" +
```

Comment:
Hmm... maybe should we use the `TransactionConfigurationException` also in the `getSender()` method? I mean, instead of returning `null` if the `signers.isEmpty()` returns true.

😆 😄 🤣 

I mean, we discussed this, but it seems that our code is not veeeery consistent. Sometimes we use exceptions, sometimes not.... we should simply agree on a pattern and stick with it.